scout/server/extensions/chanjo_extension.py

```diff
@@ -9,21 +9,22 @@
 from markupsafe import Markup
 
 LOG = logging.getLogger(__name__)
-try:
-    from chanjo_report.server.app import configure_template_filters
-    from chanjo_report.server.blueprints import report_bp
-    from chanjo_report.server.extensions import api as chanjo_api
-except ImportError as error:
-    chanjo_api = None
-    report_bp = None
-    configure_template_filters = None
-    LOG.warning("chanjo-report is not properly installed! %s.", error)
 
 
 class ChanjoReport:
     """Interfaces with chanjo-report. Creates the /reports endpoints in scout domain. Use Babel to set report language."""
 
     def init_app(self, app):
+        try:
+            from chanjo_report.server.app import configure_template_filters
+            from chanjo_report.server.blueprints import report_bp
+            from chanjo_report.server.extensions import api as chanjo_api
+        except ImportError as error:
+            chanjo_api = None
+            report_bp = None
+            configure_template_filters = None
+            LOG.error(f"{error}")
```

Comment:
The format string and variable passing is in general the preferred way to delay instantiation, but I guess no-one is going to run this at a log level below error. I can live with the eye-sore. 😁

Reply:
Ah it was a longer message before, I'll remove the format string thing.